python_modules/dagster/dagster/_core/definitions/freshness_checks/utils.py

```diff
@@ -20,11 +20,12 @@
 # Top-level metadata keys
 LAST_UPDATED_TIMESTAMP_METADATA_KEY = "dagster/last_updated_timestamp"
 FRESHNESS_PARAMS_METADATA_KEY = "dagster/freshness_params"
-# When an asset is overdue, this represents the timestamp by which the asset was expected to arrive.
-EXPECTED_BY_TIMESTAMP_METADATA_KEY = "dagster/expected_by_timestamp"
+# The latest asset record should be no earlier than this timestamp.
+LOWER_BOUND_TIMESTAMP_METADATA_KEY = "dagster/lower_bound_timestamp"
```

Comment:
I think this should be called something like "dagster/freshness_lower_bound_timestamp" to avoid conflicts with other kinds of lower bounds.

Reply:
fine with that.